src/model/model.nit

```diff
@@ -627,7 +638,7 @@ class MClassDef
 	var in_hierarchy: nullable POSetElement[MClassDef] = null
 
 	# Is the definition the one that introduced `mclass`?
-	fun is_intro: Bool do return mclass.intro == self
+	fun is_intro: Bool do return isset mclass._intro and mclass.intro == self
```

Comment:
Suggestions:
* If one is not meant to have direct low-level access to attributes ever, find another way.
* If one in meant to very exceptionally have direct low-level access to attributes, provide a keyword for it.